Add support for Ondsell

Mazuricio opened the pull request:
In Ondsel, the Path workbench is called the CAM workbench. I added parameters to find the workbench on Ondsel

Mazuricio committed:
Add support for Ondsell

Mazuricio committed:
ajust

dubstar-04 commented:
Is it worth removing the PATH references and only supporting FreeCAD 1.0 and later?

Mazuricio commented:
Sorry, I didn't understand your question. The changes only find CAM workbench (Ondsel) if it doesn't find Path workbench (Freecad), don't remove any previous support.

in this line:
`if workbench == 'PathWorkbench':`
I added an "or"
`if workbench == 'PathWorkbench' or workbench == 'CAMWorkbench' :`
and added these lines:
`if pathMenu is None:
pathMenu = mw.findChild(QtGui.QMenu, "&CAM") `

and "adjust" is a line I had deleted and duplicated the wrong one, and reverted back to the original

(Ondsel is, as described by itself: "Ondsel is built on top of the FreeCAD solid modeling engine to bring commercial features that make FreeCAD more useful to commercial users.")

dubstar-04 commented:
fixed with #57
